packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -935,7 +969,10 @@ class _RenderListTile extends RenderBox {
     final bool isOneLine = !isThreeLine && !hasSubtitle;
 
     final BoxConstraints maxIconHeightConstraint = BoxConstraints(
-      maxHeight: _maxIconHeight,
+      // One-line trailing and leading widget sizing does not follow
+      // Material specifications, but this sizing required to adhere
+      // to accessibility requirements for smallest tappable widget.
+      maxHeight: isDense ? 48.0 : 56.0,
```

Comment:
Is this constraint really only applied for one-line leading/trailing widgets?

Reply:
The constraint is applied to all types of ListTile widgets, but only one-line leading/trailing widgets will not include the Material spec padding included in its constraint, whereas the heights of two- and three-line ListTile widgets will be appropriately constrained to include Material spec padding

Reply:
A concern I may have is that this implementation could be confusing if the documentation isn't properly inspected. I'm wondering if there is some alternative approach that we could take to make the API cleaner

Reply:
I think what you've done in this PR is pretty good given the difficulty of satisfying  material's visual constraints and the accessibility constraints.